### PR TITLE
Merge from blockly/develop 07/19

### PR DESCRIPTION
**Pull request by paulkaplan**
Staying up-to-date by merging early and often! Hooray!

The larger change here is https://github.com/google/blockly/pull/1215 the refactoring of the `placeNewBlock` math.

### CODE REVIEW

File: core/workspace_svg.js [hunk 1]
```diff
@@ -340,29 +340,6 @@ Blockly.WorkspaceSvg.prototype.getOriginOffsetInPixels = function() {
 };
 
 /**
- * Return the injection div that is a parent of this workspace.
```
Comment (rachel-fenichel) [inline core/workspace_svg.js]: Why is this being deleted?

Comment (paulkaplan) [reply core/workspace_svg.js]: Woops that is a mistake. Damn now `rerere` will forever mess that one up for me :(

Comment (paulkaplan) [reply core/workspace_svg.js]: Added back in a257d49

**Comment by rachel-fenichel** [comment]
lgtm

**Pull request merged**